roles/filetree_create/templates/current_credential_types.j2

```diff
@@ -2,14 +2,23 @@
 controller_credential_types:
 {% for credential_type in current_credential_types_asset_value %}
   - name: "{{ credential_type.name }}"
-    description: "{{ credential_type.description }}"
+    description: "{{ template_overrides_resources.credential_type[credential_type.name].description
+                     | default(template_overrides_global.credential_type.description)
+                     | default(credential_type.description) }}"
     kind: "{{ credential_type.kind }}"
     inputs:
-{{ credential_type.inputs | to_nice_yaml(indent=2) | indent(width=6, first=True) }}
-{% if credential_type.injectors and credential_type.injectors != '---' %}
+{{ template_overrides_resources.credential_type[credential_type.name].inputs
+   | default(template_overrides_global.credential_type.inputs)
+   | default(credential_type.inputs) | to_nice_yaml(indent=2) | indent(width=6, first=True) }}
+{% if template_overrides_resources.credential_type[credential_type.name].injectors is defined
+      or template_overrides_global.credential_type.injectors
+      or (credential_type.injectors and credential_type.injectors != '---') %}
     injectors:
 {# https://docs.ansible.com/ansible/latest/user_guide/playbooks_advanced_syntax.html#unsafe-or-raw-strings #}
-{{ credential_type.injectors | to_nice_yaml(indent=2) | indent(width=6, first=True) | replace("'{{", "!unsafe \'{{") }}
+{{ template_overrides_resources.credential_type[credential_type.name].injectors
+   | default(template_overrides_global.credential_type.injectors)
+   | default(credential_type.injectors)
```

Comment:
Makes it sense setting all the exported `credential_types` with the same (`template_overrides_global`) injectors?

Reply:
Same as before, I'd prefer not to let doing that kind of things.